src/AutoSavings/AutoSavings.sol

```diff
@@ -49,6 +49,9 @@ contract AutoSavings is ERC7579ExecutorBase {
     // account => tokens
     mapping(address account => SentinelListLib.SentinelList) tokens;
 
+    event AccountInitialized(address indexed account);
```

Comment:
@kopy-kat shall we name it `ModuleInitialized` .. wdyt?

Reply:
yeah this is better